data/rule_data.yml

```diff
@@ -53,10 +53,6 @@ rule_data:
 
   # https://enterprisecontract.dev/docs/ec-policies/release_policy.html#labels__required_labels
   required_labels:
-  - name: architecture
-    description: Architecture the software in the image should target.
-  - name: build-date
-    description: Date/Time image was built as RFC 3339 date-time.
```

Comment:
Why do we want to remove build date?

It helps with idempotency  in the release process.

removing architecture make a sense due to to multiarch (manifest v list)

Reply:
> Why do we want to remove build date?

I wrote in the commit message - because it's redundant with existing standard metadata. The replacement for it is the `created` field in https://github.com/opencontainers/image-spec/blob/main/config.md

Reply:
```
$ skopeo inspect -n docker://registry.access.redhat.com/ubi9/ubi:latest
{
...
    "Created": "2025-01-09T06:37:11.783912772Z",
    "Labels": {
...
        "build-date": "2025-01-09T06:27:16Z",
...
```

In ubi9 for example the values are off by about 10 seconds just because they're generated at slightly different times. There's no value in the duplication.
(In fact, if one cares about [reproducible builds](https://reproducible-builds.org/) these types of random timestamps are The Enemy and need to be carefully controlled and considered)

Reply:
> In ubi9 for example the values are off by about 10 seconds (edit: minutes wow...why?) 

Because we generate the `build-date` label at the beginning of the pipeline and then pass it to the buildah task, which, I guess, adds the `Created` once it's done.
So the 10 minutes difference is the prefetch-dependencies + buildah time.
It can be slightly improved (I'll update our build pipeline) by generating the label after prefetch-dependencies, but there's still the buildah time.
+1 to removing the label.